runauto_nag.sh

```diff
@@ -19,9 +19,8 @@ python -m auto_nag.scripts.no_assignee
 # very common
 python -m auto_nag.scripts.leave_open
 
-# Try to detect potential regressions by looking at comments
-# very common
-python -m auto_nag.scripts.regression
+# Try to detect potential regressions using bugbug
+python3 -m auto_nag.scripts.regression
```

Comment:
Please also update the README.rst to add that we need pip3 too

Reply:
Fixed in ae8e658.